ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-openshift-logging-5.6.yaml

```
@@ -52,7 +52,7 @@ resources:
       memory: 4Gi
 tests:
 - as: azure-ipi-fips-f60
```

```suggestion
- as: azure-ipi-fips-f365
```

And same comment to other files.